portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -4,10 +4,12 @@
 public final class Comment {
   private String author;
   private String value;
+  private long timestamp;
 
   public Comment(String author, String value) {
     this.author = author;
     this.value = value;
+    this.timestamp = System.currentTimeMillis();
```

Comment:
I would look at java.time for getting timestamps.

Reply:
Changed.